src/instance.rs

```diff
@@ -272,7 +272,18 @@ impl<T> Py<T> {
     ///
     /// This is equivalent to the Python expression `self()`.
     pub fn call0(&self, py: Python) -> PyResult<PyObject> {
-        self.call(py, (), None)
+        cfg_if::cfg_if! {
+            // TODO: Use PyObject_CallNoArgs instead after https://bugs.python.org/issue42415,
+            // at which point can enable optimization for limited API.
```

Comment:
> at which point can enable

I'm sorry but I don't understand the grammar (caveat: I'm not a native English speaker, so please just ignore if I'm wrong)

Reply:
Yes, this is me being lazy; I've skipped the subject. I'll tidy this up 😄